nucypher/utilities/prometheus/collector.py

```diff
@@ -107,6 +107,9 @@ def initialize(self, metrics_prefix: str, registry: CollectorRegistry) -> None:
             "policies_held_gauge": Gauge(f'{metrics_prefix}_policies_held',
                                          'Policies held',
                                          registry=registry),
+           "availability_score_gauge": Gauge(f'{metrics_prefix}_availability_score',
```

Comment:
this should export as `usrula_availability_score`, just like how [3 lines above^](https://github.com/nucypher/nucypher/pull/2264/files#diff-1c60e77304346ca5724df601af97db7bL107) ends up getting exported as `ursula_policies_held` according to the [p2p-org project](https://github.com/p2p-org/nucypher-monitoring/blob/f1aed84231a4052808c492a6ff89de4f97732f94/grafana/provisioning/dashboards/nucypher.json#L1834)